src/extras/geometries/EdgesGeometry.js

```diff
@@ -86,6 +86,8 @@ THREE.EdgesGeometry = function ( geometry, thresholdAngle ) {
 
 	}
 
+	coords = coords.subarray( 0, index );
```

Comment:
Okay - that's the second of @benaadams's two suggestions. I wanted to avoid over-allocating and extra copying in my initial attempts, but, I don't think it matters too much here. I'll update the patch.